### DISCUSSION TITLE
Add Project custom fields to issue tooltip

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#108621] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I'd like to see the project custom fields in the issue tooltip, e. g. status, priority, size.
When working with "epic"/meta issues you have click every single linked issue to see this information. An addition to the tooltip would save some time and improve the user experience.

![image](https://github.com/community/community/assets/20150243/229f64f7-7db4-4def-bf09-db616302fd2a)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by medjamac** [comment]
Maxa
